extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -46,12 +47,12 @@ export class AmpExperiment extends AMP.BaseElement {
         Object.keys(config).map(experimentName => {
           return allocateVariant(this.getWin(), config[experimentName])
               .then(variantName => {
-                if (variantName) {
-                  results[experimentName] = variantName;
-                }
+                results[experimentName] = variantName;
```

Comment:
think we can return `results` here then we can get rid of `.then(() => results)`

cc @jridgewell